cluster/charts/rook-ceph/templates/_helpers.tpl

```diff
@@ -24,3 +24,10 @@ imagePullSecrets:
 {{ toYaml .Values.imagePullSecrets }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Define clusterNamespace option to use a different namespace for a ceph custer.
```

Comment:
We really need to test this with the integration tests. All of our test suites currently test with the operator in a different namespace from the cluster **except** the helm test suite. 

The ceph helm suite is defined in ceph_helm_test.go. There is a `useHelm` flag that gets passed down and eventually the function `InstallLocalRookHelmChart()` is called. This method only takes a single namespace. Now we need it to take a cluster namespace as well. Other than fixing issues that may fail in the test, this should be the main change needed so hopefully it is not too complex.